src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1791,22 +1790,13 @@ static void logEx(final String... lines) {
 		for (final String line : lines)
 			SkriptLogger.LOGGER.severe(EXCEPTION_PREFIX + line);
 	}
-	
-	public static String SKRIPT_PREFIX = ChatColor.GRAY + "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
-	
-//	static {
-//		Language.addListener(new LanguageChangeListener() {
-//			@Override
-//			public void onLanguageChange() {
-//				final String s = Language.get_("skript.prefix");
-//				if (s != null)
-//					SKRIPT_PREFIX = Utils.replaceEnglishChatStyles(s) + ChatColor.RESET + " ";
-//			}
-//		});
-//	}
-	
+
+	private static final String SKRIPT_PREFIX_DEFAULT = "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
+	private static final Message SKRIPT_PREFIX_MESSAGE = new Message("skript.prefix");
+	private static final String SKRIPT_PREFIX = SKRIPT_PREFIX_MESSAGE.getValueOrDefault(SKRIPT_PREFIX_DEFAULT);
```

Comment:
The problem with this is that the prefix will no longer update with respect to lang file changes. I would revert this change and I think the following would satisfy the other concerns:
```java
public static String getSkriptPrefix() {
	return SKRIPT_PREFIX_MESSAGE.getValueOrDefault(SKRIPT_PREFIX);
}
```
You can then call this method inside info/broadcast/etc.

You also likely want to keep SKRIPT_PREFIX_DEFAULT as SKRIPT_PREFIX and keep it public to avoid breaking changes (although it's unlikely, we could maybe check with addon tool too)

Reply:
Alright sorry took so long, head been killing me since I woke up should be changed now